engine/runtime/tests/testcase/geometry/convex_hull_accumulator.sh

```
@@ -0,0 +1,9 @@
+# /bin/bash
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)